Sync develop changes with master

0xartem opened the pull request.

0xartem committed:
fix: add ui config services available tokens

0xartem committed:
Merge pull request #122 from rsksmart/fix/ui-services-config

fix: add ui config services available tokens

nduchak reviewed (changes requested):
But should we sync develop from master instead?

nduchak committed:
Merge branch 'master' into chore/realign

nduchak committed:
fix: truffle config bug

nduchak committed:
Merge pull request #127 from rsksmart/chore/realign

Realign from master

jurajpiar committed:
refactor: renames notifications to notifier

dependabot-preview[bot] committed:
chore(deps-dev): bump truffle from 5.1.60 to 5.2.6 in /rns-dev (#123)

Bumps [truffle](https://github.com/trufflesuite/truffle) from 5.1.60 to 5.2.6.
- [Release notes](https://github.com/trufflesuite/truffle/releases)
- [Commits](https://github.com/trufflesuite/truffle/compare/truffle@5.1.60...truffle@5.2.6)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>
Co-authored-by: Nazar Duchak <41945483+nduchak@users.noreply.github.com>

0xartem committed:
fix: add default tokens whitelisting for the notifier (#131)

nduchak committed:
fix whitelisting (#129)

jurajpiar committed:
fix(rns): fixes syntax in package.json (#132)

dependabot-preview[bot] committed:
chore(deps-dev): bump truffle from 5.2.6 to 5.3.3 in /rns-dev (#135)

Bumps [truffle](https://github.com/trufflesuite/truffle) from 5.2.6 to 5.3.3.
- [Release notes](https://github.com/trufflesuite/truffle/releases)
- [Commits](https://github.com/trufflesuite/truffle/compare/truffle@5.2.6...truffle@5.3.3)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

0xartem commented:
> But should we sync develop from master instead?

there is nothing to merge from master now

dependabot-preview[bot] committed:
chore(deps-dev): bump truffle from 5.3.3 to 5.3.4 in /rns-dev (#137)

Bumps [truffle](https://github.com/trufflesuite/truffle) from 5.3.3 to 5.3.4.
- [Release notes](https://github.com/trufflesuite/truffle/releases)
- [Commits](https://github.com/trufflesuite/truffle/compare/truffle@5.3.3...truffle@5.3.4)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>
